README.md

```diff
@@ -46,11 +46,14 @@ These are some quick hints, especially for those coming from a ROS1 control back
 * <ros2_control> tags in the URDF must be compatible with the controller's configuration.
 * PLUGINLIB_EXPORT_CLASS macro is required when implementing an interface.
 
-# Test of the Scenario Before the First Release
+# Testing from the sources
 * Checkout [ros-controls/ros2_control](https://github.com/ros-controls/ros2_control) to get the core.
 * Checkout [ros-controls/ros2_controllers](https://github.com/ros-controls/ros2_controllers) to get all the controllers.
 * Checkout [ros-controls/ros2_control_demos](https://github.com/ros-controls/ros2_control_demos) to get example hardware and robot launch files.
```

Comment:
````suggestion
```
git clone https://github.com/ros-controls/ros2_control
git clone https://github.com/ros-controls/ros2_controllers
git clone https://github.com/ros-controls/ros2_control_demos
```
````
I'd simplify this with a copy-pastable instruction above